packages/theme/src/cli/utilities/theme-environment/hot-reload/error-overlay.ts

```diff
@@ -0,0 +1,74 @@
+const OVERLAY_STYLES = {
```

Comment:
I have a few opinions on the styling here but let's settle on implementation before tweaking a bunch of this!